tests/render.test.ts

```diff
@@ -23,3 +24,36 @@ test('render vite.config.js from template',  () => {
   expect(source).toMatch('viteCommonjs()')
   fs.rmdirSync(outDir, { recursive: true })
 });
+
+test('sort report by item.num', () => {
+  interface ConverObj {
+    num: string;// 'Number'
+    feat: string;// 'Conversion item'
+    times?: number;// 'Conversion times'
+  }
+  // test 1, normal
+  const reportList: ConverObj[] = []
+  const test1_nums =  ["V06","B01","B04","V01"];
+  const test1_sort = ["B01","B04","V01","V06"];
```

Comment:
Suggestion: We'd better follow the lower camel-case nomenclature
```suggestion
  const nums1 =  ["V06","B01","B04","V01"];
  const result1 = ["B01","B04","V01","V06"];
```